.gitignore

```diff
@@ -25,3 +25,7 @@ debian/pylint
 .eggs/
 .pytest_cache/
 .mypy_cache/
+venv*/
+.vscode/
```

Comment:
These are not Pylint specific but they are specific to your dev environment or development preferences. This `.gitignore` currently contains only ignores specific to the development of `pylint`.